python/ray/serve/_private/constants.py

```diff
@@ -128,6 +128,10 @@
 # The time in seconds that the Serve client waits before rechecking deployment state
 CLIENT_POLLING_INTERVAL_S: float = 1
 
+# The time in seconds that the Serve client waits before checking if
+# deployment has been created
+CLIENT_CHECK_CREATION_POLLING_INTERVAL_S: float = 0.1
```

Comment:
nit: CLIENT_CHECK_DEPLOYMENT_CREATION_POLLING_INTERVAL_S
why not use CLIENT_POLLING_INTERVAL_S?

Reply:
CLIENT_POLLING_INTERVAL_S is 1 second, which means it will add an extra 1 second delay for `serve.run` to return. Usually a deployment should be created pretty quickly (should take one controller update iteration), so I think using a poll interval of 0.1 would be better here.